content/index.mdx

```diff
@@ -4,27 +4,53 @@ title: Interface guidelines
 
 import {HeroLayout} from '@primer/gatsby-theme-doctocat'
 export default HeroLayout
-
-Primer is a set of guidelines, principles, and patterns for designing and building UI at GitHub. It provides a shared language and standardized approach to delivering cohesive experiences.
-
-#### [Guides](/guides)
-
-Standards, guidelines, and tools to getting started with Primer.
-
-#### [Foundations](/foundations)
-
-The fundamental parts of the design system that underpin all GitHub interfaces, such as color and typography.
-
-#### [UI patterns](/ui-patterns)
-
-Design guidelines covering common user workflows.
-
-#### [Components](/components)
-
-Design and usage guidelines for individual Primer components.
-
-## Need help?
-
-If you found a bug on this website, please [open a new issue](https://github.com/primer/design/issues/new) with as much detail as possible, including steps to replicate the bug, screenshots, links, and expected results.
-
-If you need help using and designing with Primer, and you are GitHub staff, please visit the #primer Slack channel.
+import {Text, Link, Box, Heading} from '@primer/react'
+import {Link as GatsbyLink} from 'gatsby'
+
+<Box mb={6}>
+  <Text as="p" sx={{m: 0, fontSize: 3}}>
+    Primer is a set of guidelines, principles, and patterns for designing and building UI at GitHub. It provides a shared language and standardized approach to delivering cohesive experiences.
+  </Text>
+</Box>
+
+<Box display="grid" gridTemplateColumns={['1fr', null, null, null, '1fr 1fr']} gridGap={5}>
+  <Box>
+    <Heading as="h3" sx={{fontSize: 3}}><Link sx={{fontSize: 3,}} as={GatsbyLink} to="/guides/">Guides</Link></Heading>
+    <Box as="p" sx={{fontSize: 2, color: 'fg.muted'}}>Standards, guidelines, and tools to getting started with Primer.</Box>
+  </Box>
+  <Box>
+    <Heading as="h3" sx={{fontSize: 3}}><Link sx={{fontSize: 3,}} as={GatsbyLink} to="/foundations/">Foundations</Link></Heading>
+    <Box as="p" sx={{fontSize: 2, color: 'fg.muted'}}>The fundamental parts of the design system that underpin all GitHub interfaces, such as color and typography.</Box>
+  </Box>
```

Comment:
Non-blocking nitpick: this could probably just be a component so we don't have to repeat ourselves so much.